theme/themes/eea/views/card.overrides

```diff
@@ -420,3 +420,29 @@
 .ui.card > a:not(.ui) > i.icon {
   color: @iconColor;
 }
+
+/*******************************
+          Teaser Card
+*******************************/
+// Teaser cards grid on tablet update padding and min-width
+@media only screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen) {
+  .ui.grid > .column.grid-block-teaser,
+  .ui.grid > .row > .column.grid-block-teaser {
+    padding-left: @tabletTeaserCardPaddingInline;
+    padding-right: @tabletTeaserCardPaddingInline;
+  }
+  .grid-block-teaser .ui.card {
+    min-width: @tabletTeaserCardMinWidth;
+  }
+}
+
+// Added from listing-cards.less
```

Comment:
Code below is taken from demo site to have the same line clamp for card header/title on storybook.
If it creates duplicate code for you we can remove it.